frontend/src/styles/footer.css

```
@@ -0,0 +1,263 @@
+.footer-bottom {
+  text-align: center;
+  padding: var(--space-lg) 0;
+  color: rgba(255, 255, 255, 0.6);
+  font-size: 0.9rem;
+}
+
+.footer-bottom p {
+  margin: var(--space-xs) 0;
+}
+
+@keyframes fadeInUp {
```

Stylelint (17.11.0)

[error] Expected keyframe name "fadeInUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)